parse-report.js

```diff
@@ -12,20 +12,26 @@ const durationRegex = /duration_ms\s([\d.]+)/
 export default async function parseReport(source) {
   const tests = []
   const testStack = []
-
-  let diagnosticMessage = ''
+  const allTests = Object.create(null)
+  let currentTest
```

Comment:
This is a little bit of a hack to hold on to the last test, so that messages that come in without file, line, and column have somewhere to go.